app/views/user_mailer/comment_notification.html.erb

```diff
@@ -0,0 +1 @@
+<h1> Hey! </h1>
```

Comment:
From a UX perspective, these email notifications are not very helpful :sweat_smile: